bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogMetadata.java

```diff
@@ -105,7 +106,7 @@ public void removeLedgerIf(LongPredicate predicate) {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("{ totalSize = ").append(totalSize).append(", remainingSize = ").append(remainingSize)
-                .append(", ledgersMap = ").append(ledgersMap).append(" }");
+                .append(", ledgersMap = ").append(StringUtils.join(ledgersMap)).append(" }");
```

Comment:
We have overridden the `toString()` method for ledgersMap. Do we need to add `join` for this map?
https://github.com/apache/bookkeeper/blob/cbb74eab4f1f464c0e4760acf9304af2289acfa1/bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongLongHashMap.java#L914-L927

Reply:
Thx for pointing it out, fixed. @hangc0276